src/main/java/ng/abdlquadri/eventbus/EventBus.java

```diff
@@ -232,17 +244,21 @@ public static void close() {
       closeFuture.addListener(new ChannelFutureListener() {
         public void operationComplete(ChannelFuture future) throws Exception {
           if (future.isSuccess() && future.isDone()){
-            log.log(Level.SEVERE, "Channel Successful Closed");
+           // silent exit is better
```

Comment:
Yes. And that's my intention. Closing should -in my opinion!- never throw an exception. It makes things worse, because often you try to close things down when something goes wrong, and when things were already closed (because they were not initialised) your closing routines gets pollinated by even more try/catches or you forget that and closing goes réally wrong. That's why I suggest to save the trouble and go for a silent exit.

You can add an extra method or callback if you really want to give this feedback.